service/application/src/main/java/com/solace/maas/ep/event/management/agent/plugin/ScanStatusPublisherDelegateImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.solace.maas.ep.event.management.agent.plugin;
+
+import com.solace.maas.ep.event.management.agent.plugin.route.RouteBundle;
+import com.solace.maas.ep.event.management.agent.plugin.route.delegate.base.MessagingServiceRouteDelegateImpl;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class ScanStatusPublisherDelegateImpl extends MessagingServiceRouteDelegateImpl {
+    public ScanStatusPublisherDelegateImpl() {
+        super("SCAN_STATUS");
+    }
+
+    @Override
+    public List<RouteBundle> generateRouteList(List<RouteBundle> destinations, List<RouteBundle> recipients,
+                                               String scanType, String messagingServiceId) {
+        return List.of(createRouteBundle(destinations, recipients, scanType, messagingServiceId,
```

Comment:
You've made sending scan status a plugin. This kind of doesn't makes sense and I think your RouteBundle builder reflects this. There's no point in this delegate class as there's no enumeration value to examine.